lib/iex/lib/iex/info.ex

```diff
@@ -160,9 +161,9 @@ defimpl IEx.Info, for: BitString do
       |> Enum.find(fn cp -> not String.valid?(cp) end)
 
     desc = """
-    This is a binary: a bunch of bytes. It's printed with the `<<>>`
-    syntax (as opposed to double quotes) because it contains non-valid
-    UTF-8 codepoints (the first non-valid codepoint being `#{inspect first_non_valid}`)
+    This is a binary: a collection of bytes. It's printed with the `<<>>`
+    syntax (as opposed to double quotes) because it is not a
+    UTF-8 encoded string (the first invalid byte being `#{inspect first_non_valid}`)
```

Comment:
string -> binary (string is redundant because a string is always utf-8 encoded for us).